tower/CHANGELOG.md

```diff
@@ -25,13 +32,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   for `ReadyCache` ([#607])
 - **steer**: Remove redundant `Req: Debug` bound from `Debug` impl
   for `Steer` ([#607])
-- **util**: Remove redundant `F: Clone` bound
-  from `ServiceExt::map_request` ([#607])
 - **docs**: Fix `doc(cfg(...))` attributes
   of `PeakEwmaDiscover`, and `PendingRequestsDiscover` ([#610])
 
 [#607]: https://github.com/tower-rs/tower/pull/607
 [#610]: https://github.com/tower-rs/tower/pull/610
+[#616]: https://github.com/tower-rs/tower/pull/616
+[#617]: https://github.com/tower-rs/tower/pull/617
+[#615]: https://github.com/tower-rs/tower/pull/615
```

Comment:
There's a part of me that desperately wants this list to be sorted... but it probably doesn't matter.

The github review UI isn't letting me leave a multiline suggestion or else i'd do that here ;)

Reply:
haha sure :P